packages/language-service/src/lib-new/lang-service-context.ts

```diff
@@ -56,6 +59,12 @@ export class LangServiceContext {
             this.fileData.content
         );
     }
+    public setFlag(flag: keyof LangServiceContext['flags'], enabled: boolean) {
```

Comment:
are we in java? I prefer not to use strings and just make the flags public.

Reply:
I felt Java and shame while writing this...